index.md

```diff
@@ -1,35 +1,20 @@
 ---
 layout: page
-title: Kì Học Bị Thiếu Của Giáo Trình Khoa Học Máy Tính
+title: The Missing Semester - kỳ học bổ sung các kiến thức hữu ích của ngành khoa học máy tính
 ---
 
-Giảng đường truyền thống dạy mọi người về các vấn đề chuyên ngành Khoa Học 
-Máy Tính cao cấp từ hệ điều hành đến học máy. Tuy nhiên có một chủ đề tối quan 
-trọng nhưng lại hay bị bỏ rơi để sinh viên tự mày mò, đó là khả năng sử dụng 
-công cụ của họ. Chúng tôi sẽ dạy bạn cách làm chủ command-line, sử dụng một trình
-biên dịch mã nguồn (text editor) hết khả năng của nó, vô vàn các chức năng "xịn xò" của trình
-quản lý phiên bản (version control systems), và hơn thế nữa.
+Các lớp học, giảng đường hiện nay  giúp bạn bổ sung kiến thức nâng cao trong ngành khoa học máy tính (CS), từ hệ điều hành đến Machine Learning. 
+Tuy nhiên, vẫn tồn tại nhiều chủ đề đóng vai trò rất quan trọng trong ngành này nhưng các giáo trình hiếm khi đề cập đến, thay vào đó, sinh viên vẫn phải tự mày mò một cách không bài bản, đó là
+kỹ năng sử dụng tốt các công cụ chuyên ngành. Trong khoá học này, chúng tôi sẽ hướng dẫn các bạn cách sử dụng thành thạo các công cụ như command-line, sử dụng một text editor có chức năng cực kỳ tốt,
+dùng các tính năng hay ho của hệ thống quản lý phiên bản mã nguồn (version control systems), và còn nhiều nội dung hữu ích khác!
```

Comment:
Please help add English (original) text below Vietnamese text as comment. You can find English texts from [here](https://github.com/missing-semester/missing-semester/edit/master/index.md).

A good format will be as follow:
```
Các lớp học...
...
... hữu ích khác!

<!--
Classes teach you all about advanced topics within CS, from operating systems
to machine learning, but there’s one critical subject that’s rarely covered,
and is instead left to students to figure out on their own: proficiency with
their tools. We’ll teach you how to master the command-line, use a powerful
text editor, use fancy features of version control systems, and much more!
-->
```

Reply:
thanks for your comments, I'll modify it

Reply:
Hi @lehoangan2906 any updates ?